src/applications/MachineShopSimulator.java

```diff
@@ -4,11 +4,11 @@
 
 public class MachineShopSimulator {
     
-    public static final String NUMBER_OF_MACHINES_MUST_BE_AT_LEAST_1 = "number of machines must be >= 1";
-    public static final String NUMBER_OF_MACHINES_AND_JOBS_MUST_BE_AT_LEAST_1 = "number of machines and jobs must be >= 1";
-    public static final String CHANGE_OVER_TIME_MUST_BE_AT_LEAST_0 = "change-over time must be >= 0";
-    public static final String EACH_JOB_MUST_HAVE_AT_LEAST_1_TASK = "each job must have >= 1 task";
-    public static final String BAD_MACHINE_NUMBER_OR_TASK_TIME = "bad machine number or task time";
+    static final String NUMBER_OF_MACHINES_MUST_BE_AT_LEAST_1 = "number of machines must be >= 1";

```

Comment:
Straight up question: Why are you changing the visibility of these? Why/how does that relate to making `Machine` less of a cata class?

Reply:
Me mindlessly accepting IntelliJ's suggestions. Has been reverted.